src/revlanguage/main.cpp

```diff
@@ -28,14 +28,38 @@ using po::variables_map;
 
 std::string usage()
 {
-    return "Usage: rb [OPTIONS]\n       rb [OPTIONS] <file1> [<file2> ...]\n";
-    // Other usages not mentioned
+    std::string usage =
```

Comment:
How about adding the other line I suggested?  From my (closed) PR:
```
rb [OPTIONS] --cmd <file> [<arg1> <arg2>...]
```

I was also thinking that maybe using lower-case for `option` like R, and dropping the angle brackets, might reduce visual clutter.  

Additionally, it might actually improve clarity to drop the square brackets around `[<arg2> ...]`.  If the user tries to use `--args` with no arguments, they will get a helpful error messages, so this is probably fine.

Lastly, we should probably indicate that you can use `--file` to add more files after arguments.

This leads to something like

```
Usage: rb [options]
       or: rb [options] file1 [file2 ...] [--args arg1 arg2 ...] [--file file3 ...]
       or: rb [options] --cmd file1 [arg1 arg2 ...] [--file file2 ...]
```
The naming of `file3` seems odd, but I'm not sure what else to write.

Reply:
Changed as suggested